templates/defaultstorageclass.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: defaultstorageclass
```

Comment:
I know it's a lame reason to block, but since this doesn't _provide_ the default storage class, can we pick a different name?

Reply:
I wouldn't call that lame, it's legitimate to want it to be more easily understood (and less easily misunderstood) at a glance. :+1: